pytorch_vs_sklearn.py

```diff
@@ -40,8 +41,30 @@
                help='where to write results to')
 p.add_argument('--seed', type=int, default=cfg.default_seed)
 p.add_argument('--verbose', action='store_true')
+
+prms = p.add_argument_group('pytorch_params')
```

Comment:
I've never seen an argument group used in argparse. I'm curious more than anything, why create a group instead of just passing the params individually? It seems like you call them the same way (e.g. args.batch_size)?

Reply:
I mostly used it because it groups the arguments together in the help output (e.g. when I call `python pytorch_vs_sklearn.py -h`), making it obvious that they have a related function.

There might be other things you can do with them, I'm not sure.